README.md

```diff
@@ -2,136 +2,130 @@
 
 ## About
```

Comment:
This is redundant. We can have the "about paragraph" directly bellow the main heading

Reply:
Done